r/README.qmd

```diff
@@ -34,3 +36,16 @@ bind_rows(
   plot(lwd = 5)
 ```
 
+
+```{r}
+x_joined_df = rnet_match(x, y, dist_tolerance = 20, slope_tolerance = 0.1, trees = "xy")
+x_aggregated_x = x_joined_df |>
+  group_by(j) |>
+  mutate(ij = y$value * shared_len) |>
```

Comment:
I think we need to subset y based on the indices in j and then multiply

Reply:
Really? Happy to see this.

Reply:
try: mutate(ij = y$value[j] * shared_len)

Because right now y$value will recycle and ignore the index of j in our result